solc/CommandLineInterface.cpp

```diff
@@ -944,7 +944,7 @@ bool CommandLineInterface::processInput()
 					"ReadFile callback used as callback kind " +
 					_kind
 				));
-			auto path = boost::filesystem::path(_path);
+			auto path = boost::filesystem::path(trimProtocol(_path));
```

Comment:
Can you do that here without defining a new function for it?

Reply:
@chriseth  sure. Done :)